### PR TITLE
Add walletconnect support

### PR DESCRIPTION
**Pull request by yarkinwho**

### CODE REVIEW

**Comment by yarkinwho** [comment]
also include fixes for minor translation issues discovered.
Including #67

File: src/i18n.js [hunk 1]
```diff
@@ -62,7 +62,33 @@ const messages = {
     'navbar.about': 'About',
     'navbar.docs': 'Docs',
     'navbar.faq': 'FAQ',
-    'footer.copyright': '© 2023 EOS Network Foundation all rights reserved.'
+    'footer.copyright': '© 2023 EOS Network Foundation all rights reserved.',
+
+    'app.walletconnect.failedswitch.title': 'Missing Network Configuration in Wallet',
+    'app.walletconnect.failedswitch.content': `<p>We've detected that the wallet you are attempting to use has not been configured to support EOS EVM yet. Please use the following network information to add the network or visit our <a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >documentation</a> page for more details.</p>
```
Comment (stephenpdeos) [inline src/i18n.js:68]: Please update to open in new tab and use the following link for EN "https://docs.eosnetwork.com/evm/quick-start/setup-metamask"

File: src/i18n.js [hunk 2]
```diff
@@ -126,7 +152,33 @@ const messages = {
     'navbar.about': '정보t',
     'navbar.docs': '문서',
     'navbar.faq': 'FAQ',
-    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.'
+    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.',
+
+    'app.walletconnect.failedswitch.title': '지갑에 네트워크 구성이 누락되었습니다',
+    'app.walletconnect.failedswitch.content': `<p>귀하가 사용하려는 지갑이 아직 EOS EVM을 지원하지 않은 것으로 확인되었습니다. 다음 네트워크 정보를 이용하여 네트워크를 추가하거나 <a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >문서 페이지</a> 를 방문하여 자세한 내용을 확인해 보시기 바랍니다.</p>
```
Comment (stephenpdeos) [inline src/i18n.js:158]: Please use the following link for KO https://docs.eosnetwork.com/ko/evm/quick-start/setup-metamask/

File: src/i18n.js [hunk 3]
```diff
@@ -190,7 +242,33 @@ const messages = {
     'navbar.about': '关于我们',
     'navbar.docs': '文档',
     'navbar.faq': '常见问题及解答',
-    'footer.copyright': '© 2023 EOS 网络基金会保留所有权利。'
+    'footer.copyright': '© 2023 EOS 网络基金会保留所有权利。',
+
+    'app.walletconnect.failedswitch.title': '钱包中缺失网络配置',
+    'app.walletconnect.failedswitch.content': `<p>我们检测到您尝试使用的钱包尚未配置以支持EOS EVM。请使用以下网络信息添加网络，或访问我们的<a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >文档页面</a>获取更多详细信息。</p>
```
Comment (stephenpdeos) [inline src/i18n.js:248]: Please use the following link for ZH https://docs.eosnetwork.com/zh/evm/quick-start/setup-metamask/

File: src/i18n.js [hunk 4]
```diff
@@ -190,7 +242,33 @@ const messages = {
...
+    <details><summary><b>Mainnet</b></summary><p dir="auto">
```
Comment (stephenpdeos) [inline src/i18n.js:249]: 主网

File: src/i18n.js [hunk 5]
```diff
@@ -190,7 +242,33 @@ const messages = {
...
+    <li>网络名称: EOS EVM</li>
+    <li>链ID: 17777</li>
+    <li>新的RPC URL: <a href="https://api.evm.eosnetwork.com/" rel="nofollow">https://api.evm.eosnetwork.com/</a></li>
+    <li>货币符号: EOS</li>
+    <li>区块浏览器URL (可选): <a href="https://explorer.evm.eosnetwork.com/" rel="nofollow">https://explorer.evm.eosnetwork.com/</a></li>
+    <li>代币桥: <a href="https://bridge.evm.eosnetwork.com/" rel="nofollow">https://bridge.evm.eosnetwork.com/</a></li>
+    </ul>
+    <p dir="auto"></p></details>
+
+    <details><summary><b>Testnet</b></summary><p dir="auto">
```
Comment (stephenpdeos) [inline src/i18n.js:260]: 测试网

File: src/i18n.js [hunk 6]
```diff
@@ -126,7 +152,33 @@ const messages = {
     'navbar.about': '정보t',
     'navbar.docs': '문서',
     'navbar.faq': 'FAQ',
-    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.'
+    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.',
+
+    'app.walletconnect.failedswitch.title': '지갑에 네트워크 구성이 누락되었습니다',
+    'app.walletconnect.failedswitch.content': `<p>귀하가 사용하려는 지갑이 아직 EOS EVM을 지원하지 않은 것으로 확인되었습니다. 다음 네트워크 정보를 이용하여 네트워크를 추가하거나 <a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >문서 페이지</a> 를 방문하여 자세한 내용을 확인해 보시기 바랍니다.</p>
+    <details><summary><b>Mainnet</b></summary><p dir="auto">
```
Comment (stephenpdeos) [inline src/i18n.js:159]: 메인넷

File: src/i18n.js [hunk 7]
```diff
@@ -126,7 +152,33 @@ const messages = {
...
+    <li>네트워크 이름: EOS EVM</li>
+    <li>체인 ID: 17777</li>
+    <li>N새 RPC URL: <a href="https://api.evm.eosnetwork.com/" rel="nofollow">https://api.evm.eosnetwork.com/</a></li>
+    <li>통화 기호: EOS</li>
+    <li>블록 탐색기 URL (옵션): <a href="https://explorer.evm.eosnetwork.com/" rel="nofollow">https://explorer.evm.eosnetwork.com/</a></li>
+    <li>토큰 브리지: <a href="https://bridge.evm.eosnetwork.com/" rel="nofollow">https://bridge.evm.eosnetwork.com/</a></li>
+    </ul>
+    <p dir="auto"></p></details>
+
+    <details><summary><b>Testnet</b></summary><p dir="auto">
```
Comment (stephenpdeos) [inline src/i18n.js:170]: 테스트넷

**Review by stephenpdeos — CHANGES_REQUESTED** [review]
A few link updates and translation updates

File: src/i18n.js (hunk 7 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 6 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 5 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 4 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 3 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 2 above)
Comment (yarkinwho) [reply src/i18n.js]: done

File: src/i18n.js (hunk 1 above)
Comment (yarkinwho) [reply src/i18n.js]: done

**Review by stephenpdeos — APPROVED** [review]
Lgtm

**Pull request merged**